factcast-snapshotcache-jdbc/src/main/java/org/factcast/core/snap/jdbc/JdbcSnapshotCache.java

```diff
@@ -168,13 +182,14 @@ String createKeyFor(SnapshotIdentifier id) {
     return ScopedName.fromProjectionMetaData(id.projectionClass()).asString();
   }
 
+  /** Updates or creates the lastAccessed timestamp if it doesn't exist or equal today's' date. */
   @VisibleForTesting
   protected void updateLastAccessedTime(@NonNull SnapshotIdentifier id) {
     try (Connection connection = dataSource.getConnection();
         PreparedStatement statement = connection.prepareStatement(updateLastAccessedStatement)) {
-      statement.setTimestamp(1, Timestamp.valueOf(LocalDate.now().atStartOfDay()));
-      statement.setString(2, createKeyFor(id));
-      statement.setString(3, id.aggregateId() != null ? id.aggregateId().toString() : null);
+      statement.setString(1, createKeyFor(id));
+      statement.setString(2, id.aggregateId() != null ? id.aggregateId().toString() : null);
+      statement.setTimestamp(3, Timestamp.valueOf(LocalDate.now().atStartOfDay()));
```

Comment:
if we go for day granularity, we should reflect that in the column type (see above)